rudder-web/src/main/webapp/secure/administration/dataSourceManagement.html

```diff
@@ -23,168 +25,209 @@ <h4 class="no-margin">
           </div>
         </div>
       </div>
-      <div class="col-md-7 col-lg-8" style="padding-left:7.5px;" ng-cloak>
+      <div class="col-xs-12 col-sm-6 col-md-7 col-lg-8" ng-cloak>
         <div class="right-container datasource-bloc col-xs-12" ng-if="selectedDatasource">
           <div class="page-title">
             <span ng-if="selectedDatasource.name.length>0">{{selectedDatasource.name}}</span>
             <span ng-if="!selectedDatasource.name && selectedDatasource.isNew">New data source</span>
             <span ng-if="!selectedDatasource.name && !selectedDatasource.isNew" style="font-style:italic; color:#777;">Unnamed</span>
           </div>
+          <!-- <h3 class="section-title">Data source information</h3> -->
           <form class="bloc-body show-details" name="forms.datasourceForm">
             <div class="form-group">
               <label for="name">Name</label>
-              <input type="text" class="form-control input-sm" id="name" ng-value="selectedDatasource.name" name="datasourceName" required ng-model="selectedDatasource.name" >
+              <input type="text" class="form-control" id="name" ng-value="selectedDatasource.name" name="datasourceName" required ng-model="selectedDatasource.name" ng-change="updateKeyName(selectedDatasource.name)">
```

Comment:
By default, this option is activated. ng-trim allows us to set it to false if necessary but it is not our case :)